docs/api/repository-signatures-resource.md

```diff
@@ -26,7 +26,7 @@ The following `@type` value is used:
 @type value                | Notes
 -------------------------- | -----
 RepositorySignatures/4.7.0 | The initial release
-RepositorySignatures/4.9.0 | Allows enabling `allRepositorySigned`
+RepositorySignatures/5.0.0 | Allows enabling `allRepositorySigned`
```

Comment:
Did an official client ship with support for 4.9.0?

If so, I personally we feel we should admit that it exists. It's a pivot available to us in the future if desired.

Reply:
Done